web/src/compose_ui.js

```diff
@@ -362,6 +363,26 @@ export function handle_keyup(_event, $textarea) {
     rtl.set_rtl_class_for_textarea($textarea);
 }
 
+export function cursor_inside_code_block($textarea) {
+    // Returns whether the cursor is at a point that would be inside
+    // a code block on rendering the textarea content as markdown.
+    const cursor_position = $textarea.caret();
+    const current_content = $textarea.val();
```

Comment:
You had just `let content` and mutated the variable. It's almost always a better coding pattern to given different variable names to the different states. I'm not sure the names I picked are ideal, but reusing the same variable name for different things (even if related) generally makes code harded to read.

Reply:
Will take care of this going forward, thanks!